doc/site/migrating-from-spring.markdown

```diff
@@ -6,43 +6,88 @@ nav_order: 5
 ---
 
 # Migrating from Spring
+{: .no_toc }
 
 While Recoil is mostly compatible with Spring 105 some divergences naturally
 developed over time.
 
 Here we list the most relevant breaking changes and how to fix them when
 migrating from Spring.
 
-## General
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
 
-- Paletted image files are no longer accepted. Convert your images not to be paletted.
-- The return value from the
-[`UnitUnitCollision`](https://beyond-all-reason.github.io/spring/ldoc/modules/LuaHandle.html#UnitUnitCollision)
-callin is now ignored and there is only one event for each collision.
-There is no way to get back the old behaviour for now,
-but if someone needs it it could be arranged.
-- Rendering fonts now obeys GL color state. This means that sometimes text
-will not be the same color as previously. To get back previous behaviour,
-you might need to add
+## Callouts
+
+### Font rendering
+
+Rendering fonts now obeys GL color state. This means that sometimes text will
```

Comment:
looks like some unrelated changes got in?

Reply:
They are a restructuring of the file. You can look at the addition of this change as a 'breaking' change on the same file.